pipeline-templates/package-vsix.yaml

```diff
@@ -30,8 +29,19 @@ jobs:
       fi
       npm version $VERSION --allow-same-version
       echo "##vso[task.setvariable variable=version;isOutput=true]$VERSION"
+
+      # Calculate if this should be prerelease based on patch version (odd = prerelease)
+      PATCH_VERSION="${VERSION##*.}"  # Gets everything after the last dot
+      IS_ODD=$((PATCH_VERSION % 2))
+      if [ $IS_ODD -eq 1 ]; then
+        echo "##vso[task.setvariable variable=isPrerelease;isOutput=true]true"
+        echo "Version $VERSION has odd patch number ($PATCH_VERSION) - will be marked as prerelease"
+      else
+        echo "##vso[task.setvariable variable=isPrerelease;isOutput=true]false"
+        echo "Version $VERSION has even patch number ($PATCH_VERSION) - will be marked as stable release"
+      fi

```

Comment:
~~So is the cadence always, even = release and odd = prerelease? What happens if there's a critical issue and needs an immediate release? Do you just skip the odd number?~~

I answered my own question by looking lower in the changes and seeing you skipped 2.3.8 so you could put out another prerelease. Got it.